Sources/ImgixSwift-Umbrella-Header.h

```diff
@@ -5,5 +5,3 @@
 //  Created by Paul Straw on 7/5/16.
 //
 //
-
-#import "NSString+ImgixSwiftMd5.h"
```

Comment:
If we remove all C/ObjC files, are we still required to provide the `ImgixSwift-Umbrella-Header.h` and `ImgixSwift.modulemap` files?